src/devices/Amg88xx/Amg88xx.cs

```diff
@@ -0,0 +1,475 @@
+        /// </summary>
+        /// <param name="frameRate">Frame rate</param>
+        /// <exception cref="ArgumentException">Thrown when attempting to set a frame rate other than 1 or 10</exception>
+        public void SetFrameRate(int frameRate)
+        {
+            if (frameRate != 1 && frameRate != 10)
+            {
+                throw new ArgumentException("Frame rate must either be 1 or 10.", nameof(frameRate));
+            }
+
+            SetBit(Register.FPSC, (byte)FrameRateBit.FPS, frameRate == 1);
+        }
+        #endregion
+
+        #region Operating Mode / Power Control
+
+        /// <summary>
+        /// Gets the current operating mode
+        /// </summary>
+        /// <returns>Operating mode</returns>
+        public OperatingMode GetOperatingMode()
+        {
+            return (OperatingMode)GetRegister(Register.PCLT);
+        }
+
+        /// <summary>
+        /// Sets the operating mode
+        /// </summary>
+        /// <param name="operatingMode">Operating mode</param>
+        public void SetOperatingMode(OperatingMode operatingMode)
+        {
+            SetRegister(Register.PCLT, (byte)operatingMode);
+        }
+        #endregion
+
+        #region Reset
+
+        /// <summary>
+        /// Performs an initial reset of the sensor. The flags and all configuration registers
+        /// are reset to default values.
+        /// </summary>
+        public void InitialReset()
+        {
+            // an initial reset (factory defaults) is initiated by writing 0x3f into the reset register (RST)
+            SetRegister(Register.RST, (byte)ResetType.Initial);
+        }
+
+        /// <summary>
+        /// Performs a reset of all flags
+        /// </summary>
+        public void FlagReset()
+        {
+            // a reset of all flags (status register, interrupt flag and interrupt table) is initiated by writing 0x30
+            // into the reset register (RST)
+            SetRegister(Register.RST, (byte)ResetType.Flag);
+        }
+
+        #endregion
+
+        #region Interrupt control, levels and pixel flags
+
+        /// <summary>
+        /// Gets the interrupt mode
+        /// </summary>
+        /// <returns>Interrupt mode</returns>
+        public InterruptMode GetInterruptMode()
+        {
+            return GetBit(Register.INTC, (byte)InterruptModeBit.INTMODE) ? InterruptMode.AbsoluteMode : InterruptMode.DifferenceMode;
+        }
+
+        /// <summary>
+        /// Sets the interrupt mode
+        /// </summary>
+        /// <param name="mode">Interrupt mode</param>
+        public void SetInterruptMode(InterruptMode mode)
+        {
+            SetBit(Register.INTC, (byte)InterruptModeBit.INTMODE, mode == InterruptMode.AbsoluteMode);
+        }
+
+        /// <summary>
+        /// Enables the interrupt pin of the AMG88xx sensor.
+        /// The pin is pulled down if an interrupt is active.
+        /// </summary>
+        public void EnableInterruptPin()
```

Comment:
Same here. `public bool InterruptPin` with the get and set function

Reply:
Done